Lib/test/test_concurrent_futures/test_thread_pool.py

```diff
@@ -67,14 +68,16 @@ def submit(pool):
                     workers.submit(tuple)
 
     @support.requires_fork()
-    @unittest.skipUnless(hasattr(os, 'register_at_fork'), 'need os.register_at_fork')
-    @support.requires_resource('cpu')
+    @unittest.skipUnless(hasattr(os, "register_at_fork"), "need os.register_at_fork")
+    @support.requires_resource("cpu")
```

Comment:
Why use different quotes?

Reply:
They seem to be random in different tests. Say, there is a test above with double quotes in decorators:
```python
    @support.requires_gil_enabled("gh-117344: test is flaky without the GIL")
    def test_idle_thread_reuse(self):
```

Though, my test is pretty similar to `test_hang_global_shutdown_lock`, so I've changed quotes to single ones